trunk/NDHMS/Routing/Reservoirs/Persistence_Level_Pool_Hybrid_Module_README.md

```diff
@@ -0,0 +1,113 @@
+type of forecast, then this parameter will automatically be set to 1,000,000,000
+seconds in order to prevent the module from reading any new timeslice observations after the first timestep.
+
+
+### Module Architecture
+
+This module builds off of the same class structure and architecture of the Level Pool module. As in Level Pool, each reservoir in this
+module is instantiated into an object at model initialization. The Persistence_Level_Pool_Hybrid directory contains
+module_persistence_levelpool_hybrid.F, module_persistence_levelpool_hybrid.F, module_persistence_levelpool_hybrid_parameters.F,
+module_persistence_levelpool_hybrid_state.F, module_persistence_levelpool_hybrid_tests.F, and a Makefile.
+
+
+The module_persistence_levelpool_hybrid.F module defines and instantiates objects for a hybrid persistence levelpool type
+reservoir. The module_RT.F module will call and pass parameters to the constructor in this module to instantiate the hybrid reservoir
+object and its sub-objects. The hybrid reservoir struct inherits input and output stucts from the reservoir base module and calls
+instantiation of these into sub-objects. The hybrid reservoir struct also points to structs for hybrid parameters and state and calls
+instantiation of these into sub-objects. A pointer to a levelpool reservoir object is also held in state, and this module
+instantiates that levelpool object. There is also a subroutine to run hybrid release that is derived from the reservoir base
+struct interface to run release. The release function will periodically call a function in module_read_timeslice_data.F that
+will read a timeslice file and return a corresponding observed discharge. The timeslice files will be read at a particular update
+time. The first hybrid reservoir on each processor to reach an update time will call the function to read the timeslice files, and
+the resulting observations will be used by each reservoir on that processor. The release function also peforms the functionality
+described above including calling level pool release along with weighting and combining that release with the weighted observed
+discharge, and finally calling mass balance checks before returning the release/outflow back to the model.
+
+The module_persistence_levelpool_hybrid_parameters.F module defines and instantiates objects for a hybrid type reservoir's
+parameters/properties. Parameters holds static/unchanging variables that are set when the given reservoir object is
+initialized/instantiated.
+
+The module_persistence_levelpool_hybrid_state.F module defines and instantiates objects for a hybrid type reservoir's state.
+State holds and tracks dynamic/changing variables that are only relevant to the given hybrid reservoir object and not other
+modules or areas of the system.
+
+The module_persistence_levelpool_hybrid_tests.F module holds unit tests that test for all components of a hybrid reservoir
+are properly initialized.
+
```

Comment:
The file summaries are nice. I'd move them up and replace the file list above with these.

Reply:
@jdmattern-noaa fixing